src/test/java/formflow/library/pdf/PdfFileTest.java

```diff
@@ -14,33 +15,93 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 class PdfFileTest {
-    PdfFile tempPdfFile;
```

Comment:
@spokenbird we seem to have different formatting rules - let's look into this when we have a chance.